charts/osdfir-infrastructure/charts/grr/README.md

```diff
@@ -0,0 +1,18 @@
+# GRR Helm Chart
+
+[GRR](https://github.com/google/grr) Rapid Response is an incident response framework focused on remote live forensics.
+
+[Overview of GRR](https://grr-doc.readthedocs.io/)
+
+Before we get started make sure you clone the repo onto your machine.
+
+```console
+git clone https://github.com/google/osdfir-infrastructure.git
+cd osdfir-infrastructure
+export REPO=$(pwd)
+
+cd charts/osdfir-infrastructure/charts/grr
```

Comment:
Is this for the docker client signing keys? Can we mention in a one-liner comment above what the script is for?

Reply:
This script is no longer required to be run as the keys are created by helm.
However, in case users would like to create keys manually then they could use the script to create them (or modify the script to match their use case).
Added a comment line to the `createSigningKeys.sh` script to indicate that it is optional:
`# Note: This is optional and just intended as an example for how keys could be created manually.`